Sources/Core/AWSClientRuntime/Tests/AWSClientRuntimeTests/Customizations/BedrockAPIKeyInterceptorTests.swift

```diff
@@ -97,7 +97,8 @@ final class BedrockAPIKeyInterceptorTests: XCTestCase {
         defer { unsetenv(envVarName) }
         let subject = BedrockAPIKeyInterceptor<String, String>()
         let context = Context(attributes: Attributes())
-        context.addIdentityResolver(value: try DefaultBearerTokenIdentityResolverChain(), schemeID: "smithy.api#httpBearerAuth")
+        let resolver = ClientConfigDefaultBearerTokenIdentityResolver(try DefaultBearerTokenIdentityResolverChain())
+        context.addIdentityResolver(value: resolver, schemeID: "smithy.api#httpBearerAuth")
```

Comment:
The bearer token identity resolver is wrapped in a ClientConfigDefaultBearerTokenIdentityResolver to signify that it was not customer-supplied.